middle_end/flambda/export_info_for_pack.ml

```diff
@@ -26,13 +26,15 @@ let imported_function_declarations_table =
 
 (* Rename export identifiers' compilation units to denote that they now
    live within a pack. *)
-let import_eid_for_pack units pack id =
+let import_eid_for_pack units prefix id =
   try Export_id.Tbl.find rename_id_state id
   with Not_found ->
     let unit_id = Export_id.get_compilation_unit id in
     let id' =
       if Compilation_unit.Set.mem unit_id units
-      then Export_id.create ?name:(Export_id.name id) pack
+      then
+        let pack = Compilation_unit.with_for_pack_prefix unit_id prefix in
```

Comment:
Same comment about below concerning the name `pack`.

Reply:
Changed to `compilation_unit`.